src/tachimawari/control/sdk/module/dynamixel_sdk.cpp

```diff
@@ -247,28 +248,34 @@ bool DynamixelSDK::sync_write_packet(
 bool DynamixelSDK::bulk_read_packet()
 {
   if (protocol_version == 1.0) {
-    sdk::GroupBulkRead group_bulk_read(port_handler, packet_handler);
-
     if (ping(CONTROLLER)) {
-      group_bulk_read.add(CONTROLLER, CM740Address::DXL_POWER, 30u);
+      group_bulk_read->add(CONTROLLER, CM740Address::DXL_POWER, 30u);
+    } else if (ping(MARIN_CORE)) {
+      group_bulk_read->add(MARIN_CORE, 64u, 20u);
     }
 
-    if (group_bulk_read.is_parameters_filled()) {
-      return send_bulk_read_packet(group_bulk_read);
-    } else {
-      return false;
+    if (group_bulk_read->is_parameters_filled()) {
+      return send_bulk_read_packet();
     }
+
+    return false;
   }
 
   return false;
 }
 
+int DynamixelSDK::get_data(
+  uint8_t id, uint16_t address, int data_length)
+{
+  return group_bulk_read->get(id, address, data_length);
+}
+
 int DynamixelSDK::get_bulk_data(
   uint8_t id, uint8_t address,
   int data_length)
 {
   if (bulk_data->find(id) != bulk_data->end()) {
-    return bulk_data->at(id).get(id, address, data_length);
+    return bulk_data->at(id)->get(id, address, data_length);
```

Comment:
can use `operator[]` instead, check other getters & setters as well as well

Reply:
I think `at` operator is better because `bulk_data` is a pointer.

Reply:
ok

Reply:
I've updated this